src/user_management/user.model.js

```diff
@@ -23,17 +23,15 @@ export default (sequelize, DataTypes) => {
       type: DataTypes.STRING,
       allowNull: false
     }
-  },
-    {
-      instanceMethods: {
-        hashPassword = (password) => {
-          return bcrypt.hash(password, bcrypt.genSaltSync(10));
-        },
-        validatePassword = (password) => {
-          return bcrypt.compare(password, this.password);
-        }
-      }
-    })
+  })
+
+  User.prototype.validPassword = function (password) {
```

Comment:
Function names must be actions. Variable names must be nouns with possible adjectives. You used a noun and an adjective for a function name 🙂 
```suggestion
  User.prototype.validatePassword = function (password) {
```